### PR TITLE
ref(js): Cleanup GSH getParams / utils

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Just some mnior refactoring + commenting

### CODE REVIEW

File: static/app/components/organizations/globalSelectionHeader/getParams.tsx [hunk 1]
```diff
@@ -82,46 +78,44 @@ const normalizeDateTimeString = (
...
 
-const getUtcValue = (
-  maybe: string | string[] | boolean | undefined | null
-): string | undefined => {
-  if (Array.isArray(maybe)) {
-    if (maybe.length <= 0) {
-      return undefined;
-    }
+/**
+ * Normalizes a sring or string[] into the a date time string.
```
Comment (matejminar) [inline static/app/components/organizations/globalSelectionHeader/getParams.tsx:84]:
```suggestion
 * Normalizes a string or string[] into the date time string.
```

File: static/app/components/organizations/globalSelectionHeader/getParams.tsx [hunk 2]
```diff
@@ -17,44 +23,31 @@ export function parseStatsPeriod(input: string | IntervalPeriod) {
...
-
-  return `${period}${periodLength}`;
+  return result ? `${result.period}${result.periodLength}` : undefined;
 }
 
-function getStatsPeriodValue(
-  maybe: string | string[] | undefined | null
-): string | undefined {
+/**
+ * Noramlizes a string or string[] into a standard stats period string.
```
Comment (matejminar) [inline static/app/components/organizations/globalSelectionHeader/getParams.tsx:42]:
```suggestion
 * Normalizes a string or string[] into a standard stats period string.
```

File: static/app/components/organizations/globalSelectionHeader/getParams.tsx [hunk 3]
```diff
@@ -17,44 +23,31 @@ export function parseStatsPeriod(input: string | IntervalPeriod) {
...
 
-  return {
-    period,
-    periodLength,
-  };
+  return {period, periodLength};
 }
 
+/**
+ * Noramalizes a stats period string
```
Comment (matejminar) [inline static/app/components/organizations/globalSelectionHeader/getParams.tsx:33]:
```suggestion
 * Normalizes a stats period string
```

**Review by matejminar — APPROVED** [review]

File: static/app/components/organizations/globalSelectionHeader/getParams.tsx [hunk 4]
```diff
@@ -64,13 +57,16 @@ function getStatsPeriodValue(
...
-// We normalize potential datetime strings into the form that would be valid
-// if it were to be parsed by datetime.strptime using the format %Y-%m-%dT%H:%M:%S.%f
-// This format was transformed to the form that moment.js understands using
-// https://gist.github.com/asafge/0b13c5066d06ae9a4446
-const normalizeDateTimeString = (
-  input: Date | string | undefined | null
-): string | undefined => {
+/**
+ * We normalize potential datetime strings into the form that would be valid if
+ * it were to be parsed by datetime.strptime using the format
```
Comment (priscilawebdev) [inline static/app/components/organizations/globalSelectionHeader/getParams.tsx:62]:
```suggestion
 * it was to be parsed by datetime.strptime using the format
```

**Review by priscilawebdev — APPROVED** [review]
with the typos fixes, it looks good to me

**Pull request merged**